tasks/Resources/RHODS_OLM/post-tasks/oc_version.robot

```diff
@@ -0,0 +1,7 @@
+*** Keywords ***  
+Get RHODS Version
```

Comment:
is there another keyword which gets the RHODS versions at this path: `tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot`. Are these equivalent? Should we use only one of them?

Reply:
The idea was to separate the tasks with the tests to avoid problems running the tests, so I had to create get RHODS version for tasks, given that we are not using the same resources I think that it won't create a keyword conflict. Do you think that it would be necessary to modify the keyword name?

Reply:
mmmh got your point.I would leave it as you did, and then we could discuss about this kind of situation during refactoring to take a common decision. What do you think?